src/components/CityWeatherCard.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styles from './CityWeatherCard.module.css';
+
+export interface CityWeatherCardProps {
+    name: string;
+    temperature: string;
+    conditions: string;
+    wind_speed: string;
```

Comment:
Even though the data is coming back with snake case attributes in the JSON, we should stick to a single convention in our code (the standard on this is [lowerCamelCase](https://en.wikipedia.org/wiki/Camel_case). `lowerCamelCase`, `UpperCamelCase`. with some exceptions.

For variables and the default, would recommend using `lowerCamelCase`.
For class names, types and components names, would recommend using `UpperCamelCase`.